scripts/plot_performance_breakdown.py

```diff
@@ -6,14 +6,18 @@
 import json
 import matplotlib.pyplot as plt
 import pandas as pd
-import sys
+import argparse
```

Comment:
depends on the usecase i guess, but would be no effort to add a --optimizer_only flag.

Reply:
What is the advantage of having two graphs or being able to disable a subplot?

Reply:
i went for an optional seperate plot without the non-optimizer steps. exporting subplots seems cumbersome and since sharing axes is not very helpful in our case i decided against subplots.

Reply:
In that case, I'd simply plot it in a second file, skipping the need for a CLI parameter. Not sure what that parameter adds. @Bensk1?

Reply:
i we have the optimizer in a seperate plot, i think it an extra parameter doesn't add that much. removed it.